packages/retail-ui-extensions/src/components/TimeField/TimeField.ts

```diff
@@ -0,0 +1,21 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import {InputProps} from 'components/shared';
+
+export interface TimeFieldProps
+  extends Pick<
```

Comment:
same here: also need `helpText` and `action` which are POS specific additional props